app/controllers/concerns/idv/phone_question_ab_test_concern.rb

```diff
@@ -0,0 +1,29 @@
+module Idv
+  module PhoneQuestionAbTestConcern
+    def phone_question_ab_test_bucket
+      AbTests::IDV_PHONE_QUESTION.bucket(phone_question_user.uuid)
+    end
+
+    def phone_question_user
+      if defined?(document_capture_user) # hybrid flow
+        document_capture_user
+      else
+        current_user
+      end
+    end
+
+    def maybe_redirect_for_phone_question_ab_test
+      return if phone_question_ab_test_bucket != :show_phone_question
+      return if request.referer == idv_phone_question_url
```

Comment:
When I clicked the 'Back' link on LinkSent, I got sent to PhoneQuestion. Debatable whether that's useful to the user, but I expected HybridHandoff. If you want it to go to HybridHandoff, add
`return if request.referer == idv_link_sent_url`

Reply:
✅